fil-proofs-tooling/Cargo.toml

```diff
@@ -42,5 +42,5 @@ log = "0.4.8"
 uom = "0.25.0"
 
 [features]
-default = []
+default = ["gpu"]
```

Comment:
By making this the default, does it mean a GPU is required? Or that it'll be used by default if detected?

Reply:
it means it well run the detection and use it if it finds a compatible one